src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6007,7 +6007,7 @@ export class StudyViewPageStore {
                         )
                     );
                     // samples countaing this data would be the samples profiled for these molecular profiles
-                    ret[uniqueKey] = _.sumBy(
+                    let count = _.sumBy(
```

Comment:
logic in this if block seems odd.  i think you can put the uniqueKey logic at 6004 in the else block of count===0 since you otherwise don't use it.